### PR TITLE
Pending Document IDs API new error message

### PR DESCRIPTION
**Pull request by Sandychuang8**

### CODE REVIEW

**Comment by Sandychuang8** [comment]
Please remember to add this error message in the new Pending Document IDs PR.

File: gen_errors/cbl-errors.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     { "name": "DuplicateSelectResultName", "message": "Duplicate select result named {0}." },
     { "name": "NoAliasInJoin", "message": "The default database must have an alias in order to use a JOIN statement (Make sure your data source uses the As() function)." },
     { "name": "InvalidQueryDBNull", "message": "Invalid query: The database is null." },
-    { "name": "InvalidQueryMissingSelectOrFrom", "message": "Invalid query: missing Select or From." }
+    { "name": "InvalidQueryMissingSelectOrFrom", "message": "Invalid query: missing Select or From." },
+    { "name": "PullOnlyReplicatorsNotSupportPendingDocIds", "message": "Pending Document IDs are not supported on pull-only replicators." }
```
Comment (jayahariv) [inline gen_errors/cbl-errors.json]: suggestion on key, `PullOnlyReplicatorsNotSupportPendingDocIds`, seems complicated and difficult to grasp the meaning. 

how about `UnsupportedReplicatorForPendingDoc`

Comment (Sandychuang8) [reply gen_errors/cbl-errors.json]: Hmm, that sounds strange... it sounds like replicator is not supported for pending doc...

Comment (Sandychuang8) [reply gen_errors/cbl-errors.json]: @borrrden suggests `PullOnlyPendingDocIDs`. I think it's quite fine.

**Pull request merged**